app/tracking/get-scheme-id.js

```diff
@@ -0,0 +1,48 @@
+const schemes = require('../constants/schemes')
```

Comment:
You could import these as `{ SFI, SFIP... }` to avoid needing to prefix everything with `scheme.`

Reply:
I've resolved this by making them lowercase - not ideal, but it works